### DISCUSSION TITLE
Lists with a cloud variable icon?

### DISCUSSION THREAD

**Discussion by LeafiDev** [TurboWarp/packager#290] [Q&A]
We can put clouds in front of lists and for some reason we can't edit those in the packager. or is there a way?
![image](https://user-images.githubusercontent.com/83046401/152085354-91a51692-64ba-4068-bc29-c517765e30bf.png)
(they actually list themselves off as cloud registered.)

**Comment by GarboMuffin** [comment]
Cloud lists aren't supported.
